k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -1616,4 +1616,9 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         }
     }
+
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
```

Comment:
You misunderstand. Adding `orientation` to the AndroidManifest prevents the activity being restarted. I'm asking why we need to override this method. If you remove this override nothing will change - the activity still won't be restarted. We're not doing anything here.